src/Module.Server/Modes/Dtv/CrpgDtvSpawningBehavior.cs

```diff
@@ -132,6 +136,7 @@ private void SpawnAttackers(CrpgDtvWave wave, int defendersCount)
         {
             var group = groupsWithoutBoss[i % groupsWithoutBoss.Length];
             SpawnBotAgent(group.ClassDivisionId, Mission.AttackerTeam);
+            SpawnedAttackers++;
```

Comment:
This seems to be not used?

Reply:
I've removed the unused counter & timer. They were added as I was planning to remove delaying cavalry - which is no longer being implemented.